tests/load/pipeline/test_merge_disposition.py

```diff
@@ -30,13 +33,30 @@
 # ACTIVE_DESTINATIONS += ["motherduck"]
 
 
+def skip_if_not_supported(
+    merge_strategy: TLoaderMergeStrategy,
+    destination: TDestination,
+) -> None:
+    if merge_strategy not in destination.capabilities().supported_merge_strategies:
```

Comment:
this is good. but should be a part of `destinations_configs` filters. definitely a task for the future: if we add supported write disposiitons to the capabilities we'll be able to generate a lot of filters automatically